Kitodo-DataManagement/src/main/resources/db/migration/V2_121__Add_selectable_audio_video_preview_and_mediaview_folders_to_project.sql

```diff
@@ -0,0 +1,30 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+-- Add audio and video preview and media view folder to project
+
+ALTER TABLE project
+  ADD preview_video_folder_id int(11) DEFAULT NULL
+        COMMENT 'media to use for video gallery preview',
+  ADD CONSTRAINT CK_preview_video_folder_id
```

Comment:
Should it not be `FK_preview_video_folder_id` instead of current `CK_preview_video_folder_id`  where `FK` is standing for "foreign key"?  If this `FK` is correct then it should be changed in the whole file.